examples/openapi3/basicauth/app.py

```diff
@@ -4,29 +4,16 @@
 '''
 
 import connexion
-from connexion.decorators.security import validate_scope
-from connexion.exceptions import OAuthScopeProblem
 
+PASSWD={
+    'admin': 'secret',
+    'foo': 'bar'
+}
 
-def basic_auth(username, password, required_scopes=None):
-    if username == 'admin' and password == 'secret':
-        info = {'sub': 'admin', 'scope': 'secret'}
-    elif username == 'foo' and password == 'bar':
-        info = {'sub': 'user1', 'scope': ''}
-    else:
-        # optional: raise exception for custom error response
```

Comment:
nit but can we leave this comment in here to indicate that this is also possible to do from a security function?